src/main/java/com/contrastsecurity/maven/plugin/AbstractContrastMavenPluginMojo.java

```diff
@@ -37,9 +35,12 @@
     @Parameter(property = "orgUuid", required = true)
     protected String orgUuid;
 
-    @Parameter(property = "appName", required = true)
+    @Parameter(property = "appName")
```

Comment:
Can you see if there is there any way to declare that one parameter or another is required? appName is no longer required, but either appName or appId is required.

Reply:
Looks like there is no way to require one of two parameters, but I added an exception in case no parameter is specified.